projects/plugins/jetpack/modules/custom-css/csstidy/class.csstidy_optimise.php

```diff
@@ -546,393 +586,422 @@ static function dissolve_4value_shorthands($property, $value) {
-				$return['font-style'] = $str_value[0][$j];
-				$have['style'] = true;
-			} elseif ($have['size'] === false && (is_numeric($str_value[0][$j][0]) || $str_value[0][$j][0] === null || $str_value[0][$j][0] === '.')) {
-				$size = csstidy_optimise::explode_ws('/', trim($str_value[0][$j]));
+		// Workaround with multiple font-family.
+		$str_value = self::explode_ws( ',', trim( $str_value ) );
+
+		$str_value[0] = self::explode_ws( ' ', trim( $str_value[0] ) );
+
+		for ( $j = 0; $j < count( $str_value[0] ); $j++ ) { // phpcs:ignore Squiz.PHP.DisallowSizeFunctionsInLoops.Found, Generic.CodeAnalysis.ForLoopWithTestFunctionCall.NotAllowed
+			if ( $have['weight'] === false && in_array( $str_value[0][ $j ], $font_weight ) ) { // phpcs:ignore WordPress.PHP.StrictInArray.MissingTrueStrict
+				$return['font-weight'] = $str_value[0][ $j ];
+				$have['weight']        = true;
+			} elseif ( $have['variant'] === false && in_array( $str_value[0][ $j ], $font_variant, true ) ) {
+				$return['font-variant'] = $str_value[0][ $j ];
+				$have['variant']        = true;
+			} elseif ( $have['style'] === false && in_array( $str_value[0][ $j ], $font_style, true ) ) {
+				$return['font-style'] = $str_value[0][ $j ];
+				$have['style']        = true;
+			} elseif ( $have['size'] === false && ( is_numeric( $str_value[0][ $j ][0] ) || $str_value[0][ $j ][0] === null || $str_value[0][ $j ][0] === '.' ) ) {
+				$size                = self::explode_ws( '/', trim( $str_value[0][ $j ] ) );
 				$return['font-size'] = $size[0];
-				if (isset($size[1])) {
+				if ( isset( $size[1] ) ) {
 					$return['line-height'] = $size[1];
 				} else {
 					$return['line-height'] = ''; // don't add 'normal' !
 				}
 				$have['size'] = true;
 			} else {
-				if (isset($return['font-family'])) {
-					$return['font-family'] .= ' ' . $str_value[0][$j];
-					$multiwords = true;
+				if ( isset( $return['font-family'] ) ) {
+					$return['font-family'] .= ' ' . $str_value[0][ $j ];
+					$multiwords             = true;
 				} else {
-					$return['font-family'] = $str_value[0][$j];
+					$return['font-family'] = $str_value[0][ $j ];
 				}
 			}
 		}
-		// add quotes if we have several qords in font-family
-		if ($multiwords !== false) {
+		// add quotes if we have several qords in font-family.
+		if ( $multiwords !== false ) {
 			$return['font-family'] = '"' . $return['font-family'] . '"';
 		}
 		$i = 1;
-		while (isset($str_value[$i])) {
-			$return['font-family'] .= ',' . trim($str_value[$i]);
+		while ( isset( $str_value[ $i ] ) ) {
+			$return['font-family'] .= ',' . trim( $str_value[ $i ] );
 			$i++;
 		}
 
-		// Fix for 100 and more font-size
-		if ($have['size'] === false && isset($return['font-weight']) &&
-						is_numeric($return['font-weight'][0])) {
+		// Fix for 100 and more font-size.
+		if ( $have['size'] === false && isset( $return['font-weight'] ) &&
+						is_numeric( $return['font-weight'][0] ) ) {
```

Comment:
I'm surprised phpcs didn't complain about this.
```suggestion
		if ( $have['size'] === false && isset( $return['font-weight'] ) &&
			is_numeric( $return['font-weight'][0] )
		) {
```